### ISSUE TITLE
PHP 7 deprecated errors on phpseclib

### ISSUE THREAD

**Issue by allanfreitas** [rocketeers/rocketeer#628]
Hi, I'm getting this warnings:
```php
$ rocketeer deploy --on=production
production/0 | Deploy (Deploys the website)

Deprecated: Methods with the same name as their class will not be constructors in a future version of PHP; Net_SFTP has a deprecated constructor in C:\Users\MYSER\AppData\Roaming\Composer\vendor\phpseclib\phpseclib\phpseclib\Net\SFTP.php on line 122

Deprecated: Methods with the same name as their class will not be constructors in a future version of PHP; Net_SSH2 has a deprecated constructor in C:\Users\MYSER\AppData\Roaming\Composer\vendor\phpseclib\phpseclib\phpseclib\Net\SSH2.php on line 151

Deprecated: Methods with the same name as their class will not be constructors in a future version of PHP; Math_BigInteger has a deprecated constructor in C:\Users\MYSER\AppData\Roaming\Composer\vendor\phpseclib\phpseclib\phpseclib\Math\BigInteger.php on line 176

Deprecated: Methods with the same name as their class will not be constructors in a future version of PHP; Crypt_Hash has a deprecated constructor in C:\Users\MYSER\AppData\Roaming\Composer\vendor\phpseclib\phpseclib\phpseclib\Crypt\Hash.php on line 82

Deprecated: Methods with the same name as their class will not be constructors in a future version of PHP; Crypt_RC4 has a deprecated constructor in C:\Users\MYSER\AppData\Roaming\Composer\vendor\phpseclib\phpseclib\phpseclib\Crypt\RC4.php on line 101

Deprecated: Methods with the same name as their class will not be constructors in a future version of PHP; Crypt_Base has a deprecated constructor in C:\Users\MYSER\AppData\Roaming\Composer\vendor\phpseclib\phpseclib\phpseclib\Crypt\Base.php on line 121
production/0 |-- Primer (Run local checks to ensure deploy can proceed)
production/0 |-- CreateRelease (Creates a new release on the server)
production/0 |---- Deploy/Clone (Clones a fresh instance of the repository by SCM)
production/0 |===> Cloning repository in "/apps/MYAPP/releases/20160107195429"
```

I've checked on the phpseclib and it's already fixed on 2.0 version, on this issue https://github.com/phpseclib/phpseclib/issues/734

**Comment by mbrodala** [comment]
This should be fixed since 1f11249

**Comment by Anahkiasen** [comment]
Yup
